circus-train-s3-mapreduce-cp/src/main/java/com/hotels/bdp/circustrain/s3mapreducecp/mapreduce/RetriableFileCopyCommand.java

```diff
@@ -133,10 +134,9 @@ private long doCopy(Mapper.Context context, FileStatus sourceFileStatus, S3Uploa
   }
 
   private static ThrottledInputStream getInputStream(Path path, Configuration conf) throws IOException {
-    try {
-      FileSystem fs = path.getFileSystem(conf);
-      long bandwidthMB = conf.getInt(ConfigurationVariable.MAX_BANDWIDTH.getName(),
-          ConfigurationVariable.MAX_BANDWIDTH.defaultIntValue());
+    try (FileSystem fs = path.getFileSystem(conf)) {
```

Comment:
We use `FileSystem` all over the place and we never call close I think, is this the only place where checkstyle is complaining about this, is that because of the `fs.open` call on line 140? Getting the FS from the path might involve getting a cached one and closing it would remove it from the cache I don't think closing it is necessary in our case.

Reply:
Cool, I'll undo this then :)